### DISCUSSION TITLE
Is it possible to hide the commit field?

### DISCUSSION THREAD

**Discussion by therealpoys** [netbox-community/netbox#9580] [Q&A] [answered]
Hello,
it possible to hide the commit field in custom scripts?
Your sincerely,
Jannik

**Comment by jeremystretch** [comment, marked as answer]
You cannot hide the field, as it's important for the user to understand the implications of running a script, however you can [tweak its default value](https://docs.netbox.dev/en/stable/customization/custom-scripts/#commit_default).

**Reply by samk-acw** [reply]
Shouldn't that be up to the discretion of the script author? Not all scripts might make changes - we have several that are used for status checks and lookups that don't modify anything, so the commit box adds some confusion to unaware users.

**Comment by mtinberg** [comment]
Aren't Netbox Reports a better fit for the case of not making changes?  They operate almost identically to Scripts but are not intended to commit.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________

**Reply by therealpoys** [reply]
Reports can't have user inputs. If reports can have a user input, for something like a keyword to search, then we can use the reports.
